registry/tracecat_registry/templates/detect/lookup_domain/threatstream.yml

```
@@ -0,0 +1,31 @@
+type: action
+definition:
+  name: lookup_domain
+  namespace: tools.threatstream
+  title: Lookup Domain
+  description: Search Anomali ThreatStream for reputation of a domain
+  display_group: Anomali ThreatStream
+  secrets:
+    - name: threatstream
+      keys:
+        - USERNAME
+        - API_KEY
+  expects:
+    domain:
+      type: str
+      description: Domain to lookup.
+  steps:
+    - ref: make_request

```

```suggestion
    - ref: get_intelligence_report
```

Convention is to name the step similar to how the official API docs name it (e.g. "Get threat intel report"). Wonder if there are any public docs for ThreatStream as well to populate `doc_url` under display group?

---

No public docs for threatstream, otherwise would have linked.  Indicator lookup is different than pulling a threat report.  This is more of a reputation check.

---

Updated all the step refs to be named get_reputation.